media/js/src/AnnotationScroller.jsx

```diff
@@ -37,7 +56,9 @@ export default class AnnotationScroller extends React.Component {
                 currentAnnotation: newAnnotation
             });
 
-            this.props.onSelectedAnnotationUpdate(newAnnotation);
```

Comment:
Removing `onSelectedAnnotationUpdate` prevents the annotation view from updating. Also, the behavior between onNextClick and onPrevClick should be consolidated into a single block of logic. Current state of this with these changes are inconsistend - I'm not sure if this is still in progress though.

Reply:
@nikolas definitely deleted by accident

The idea behind the logic is if that the alt label would change according to the annotation but at one point the on prev click would reach the original asset which would change the wording of the alt to identify it as an Asset and not an Annotation.